ludwig/model_export/base_model_exporter.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
+
+import torch
+
+from ludwig.api import LudwigModel
+
+
+class LudwigTorchWrapper(torch.nn.Module):
+    def __init__(self, model):
+        super().__init__()
+        self.model = model
+
+    def forward(self, x):
+        return self.model({"image_path": x})
+
+
+class BaseModelExporter(ABC):
```

Comment:
Could we add docstrings to these new classes?

Reply:
Added